### ISSUE TITLE
I Misunderstood Breadcrumbs 

### ISSUE THREAD

**Issue by KrisSimon** [statamic/cms#10401]
### Bug description

A breadcrumb is a vertical navigation that reflects the shortest way to the current page from home. So it is completely based on the navigation of a homepage. 
When I do a nav:breadcrumbs tag, the structure is taken from the pages collection, that could be technical and does not reflect the page  structure of the resulted page. 

Why this is so, and how to I get the navigation path to the current page?

### How to reproduce

```
<ul class="breadcrumbs">
    {{ nav:breadcrumbs include_home=false }}
    <li{{ if is_current }} class="current"{{ /if }}>
        <a href="{{ url }}">{{ title }}</a>
    </li>
    {{ /nav:breadcrumbs }}
</ul>
```
is the structure of the technical location of a page, not the navigation. 

### Environment

```yaml
PHP 8.2.18
Statamic: 4.58.2 Pro
```

### Installation

Fresh statamic/statamic site via CLI

**Comment by jasonvarga** [comment]
The breadcrumbs tag is based on the URL structure.

If you're at `/blog/post-1` it'll output
- `/blog/post-1` which could be in the `blog` collection
- then `/blog` which is probably in the `pages` collection
- then `/` which is also probably in the `pages` collection

It doesn't matter if they are in the same collection.

**Comment by KrisSimon** [comment]
So far so good, by why is the  breadcrumb and the URL structure based on the order of the entries in the collections and not on the navigation. 

Eg. I have some pages under /marketing, some under /news. In my navigation I set my navigation layout that ist for example - - /useful-links/marketing12
- /useful-links/marketing1
- /useful-links/news100
- /useful-links/marketing2

If I the the BCN, the url is `/` > `marketing` > `marketing 12`, I would rarther expect the navigation path: `/` marketing  `useful-links` > `merketing12`. 

Why do I have a navigation if this is not reflected in the URL schema, nor in the BCN. 
Then I could order my pages in the collection like I want them to be to appear on my final site and do not need a navigation at all. 
When I build my site with a navigation, this is the truth for all outside views and should reflect the page tree in all belongings.

**Comment by KrisSimon** [comment]
Let me rephrase my question: How can I get the navigation path down to the site I am on?

**Comment by jasonvarga** [comment]
Sorry I don't really understand what you're asking.

A navigation is just a compilation of entries/pages/etc. It's not intended to modify the URL structure of anything.
